graphs/possible_bipartition.py

```diff
@@ -1,12 +1,32 @@
 # Can be used for BFS
-from collections import deque 
+from collections import deque
 
 def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: the worst, O(n^2)
+        Space Complexity: O(n)
     """
-    pass
-
+    if len(dislikes) == 0:
+        return True
+    
+    group1 = set()
+    group1.add(0)
```

Comment:
On line 19, if you end up skipping over the 0th node since it has no neighbors and iterating over the 1st node, you may want to consider adding node 1 to group1 to start.

Otherwise you end up only adding node 1 to a set if it is disliked by a different dog.